server/webhook_parser.go

```diff
@@ -316,6 +318,77 @@ func quoteIssueComment(comment string) string {
 	return "> " + strings.ReplaceAll(comment, "\n", "\n> ")
 }
 
+func preProcessText(comment string) string {
```

Comment:
Add a comment to explain the use of this function. You can also add the issue link for reference